src/mpid/ch4/netmod/ofi/ofi_impl.h

```diff
@@ -38,7 +38,7 @@
  */
 #define MPIDI_OFI_PROGRESS()                                      \
     do {                                                          \
-        mpi_errno = MPID_Progress_test();                        \
+        mpi_errno = MPIDI_NM_progress(0, 0);                      \
```

Comment:
Are `FI_EAGAIN` errors guaranteed to eventually complete if you keep retrying?

Reply:
Hmm, it should give up after a certain number of trials as other parts in OFI netmod. I'll update this.